python-sdk/nuscenes/lidarseg/lidar_dataviz.py

```diff
@@ -0,0 +1,197 @@
+        "vehicle.bus.bendy": 'bus.bendy',
+        "vehicle.bus.rigid": 'bus.rigid',
+        "vehicle.truck": 'truck',
+        "vehicle.construction": 'constr. veh',
+        "vehicle.emergency.ambulance": 'ambulance',
+        "vehicle.emergency.police": 'police car',
+        "vehicle.trailer": 'trailer',
+        "movable_object.barrier": 'barrier',
+        "movable_object.trafficcone": 'trafficcone',
+        "movable_object.pushable_pullable": 'push/pullable',
+        "movable_object.debris": 'debris',
+        "static_object.bicycle_rack": 'bicycle racks',
+        "flat.driveable_surface": 'driveable',
+        "flat.sidewalk": 'sidewalk',
+        "flat.terrain": 'terrain',
+        "flat.other": 'flat.other',
+        "static.manmade": 'manmade',
+        "static.vegetation": 'vegetation',
+        "static.other": 'static.other',
+        "vehicle.ego": "ego"
+    }
+
+    return string_mapper[class_name]
+
+
+def render_lidarseg_histogram(nusc: NuScenes,
+                              sort_by: str = 'count',
+                              chart_title: str = None,
+                              x_label: str = None,
+                              y_label: str = "Lidar points (logarithmic)",
+                              y_log_scale: bool = True,
+                              verbose: bool = True,
+                              font_size: int = 20,
+                              save_as_img_name: str = None) -> None:
+    """
+    Render a histogram for the given nuScenes split.
+    :param nusc: A nuScenes object.
+    :param sort_by: How to sort the classes:
+        - count_desc: Sort the classes by the number of points belonging to each class, in descending order.
+        - count_asc: Sort the classes by the number of points belonging to each class, in ascending order.
+        - name: Sort the classes by alphabetical order.
+        - index: Sort the classes by their indices.
+    :param chart_title: Title to display on the histogram.
+    :param x_label: Title to display on the x-axis of the histogram.
+    :param y_label: Title to display on the y-axis of the histogram.
+    :param y_log_scale: Whether to use log scale on the y-axis.
+    :param verbose: Whether to display plot in a window after rendering.
+    :param font_size: Size of the font to use for the histogram.
+    :param save_as_img_name: Path (including image name and extension) to save the histogram as.
+    """
+
+    print('Calculating stats for nuScenes-lidarseg...')
+    start_time = time.time()
+
+    # Get the statistics for the given nuScenes split.
+    class_names, counts = get_lidarseg_stats(nusc, sort_by=sort_by)
+
+    print('Calculated stats for {} point clouds in {:.1f} seconds.\n====='.format(
+        len(nusc.lidarseg), time.time() - start_time))
+
+    # Place the class names and counts into a dataframe for use with seaborn later.
+    df = pd.DataFrame(list(zip(class_names, counts)), columns=['Class', 'Count'])
```

Comment:
Let's not use pandas here, as it is not a base requirement

Reply:
Found workaround to avoid using pandas